packages/runtime/test/transport/messages.test.ts

```diff
@@ -272,6 +275,73 @@ describe("Message errors", () => {
         expect(result).toBeAnError("The recipient does not match the Request's peer.", "error.runtime.validation.invalidPropertyValue");
     });
 
+    test("should throw correct error message for trying to send a Message with a Request that expired", async () => {
+        const requestItem = {
+            "@type": "ConsentRequestItem",
+            consent: "I consent to this RequestItem",
+            mustBeAccepted: true
+        };
```

Comment:
There isn't a need to define this `requestItem` again as it is already defined within the `beforeAll` block of the tests.